python/test/test_server.py

```diff
@@ -1,660 +1,382 @@
+            self.request('CreateDataObject', data_object=data_obj)
+        # Make sure that the data objects we just created exist
+        for id_ in ids:
+            self.request('GetDataObject', data_object_id=id_)
+
+        # Mint a data bundle with the data objects we just created
+        data_bundle = self.generate_data_bundle(data_object_ids=ids)
+        self.request('CreateDataBundle', data_bundle=data_bundle)
+        # Does the data bundle we just created exist?
+        server_bdl = self.request('GetDataBundle', data_bundle_id=data_bundle.id).data_bundle
+        self.assertSameDataBundle(server_bdl, data_bundle)
 
         # UpdateDataBundle
-        print("..........Update that Bundle.................")
-        UpdateDataBundleRequest = self._models.get_model(
-            'UpdateDataBundleRequest')
-        update_data_bundle = DataBundle(
-            id=str(uuid.uuid1()),
-            name="abc",
-            size="12345",
-            created=datetime.utcnow(),
-            updated=datetime.utcnow(),
-            version=str(datetime.utcnow()),
-            data_object_ids=[x.id for x in list_response.data_objects],
-            checksums=[Checksum(checksum="def", type="md5")],
-            aliases=["ghi"])
-        update_request = UpdateDataBundleRequest(data_bundle=update_data_bundle)
-        update_response = self._client.UpdateDataBundle(
-            data_bundle_id=data_bundle_id,
-            body=update_request).result()
-        print("..........Update that Bundle.................")
-        updated_bundle = self._client.GetDataBundle(
-            data_bundle_id=update_response['data_bundle_id'])\
-            .result().data_bundle
-        print('updated_bundle.aliases', updated_bundle.aliases)
-        print('updated_bundle.updated', updated_bundle.updated)
-        print('data_bundle.aliases', data_bundle.aliases)
-        print('data_bundle.updated', data_bundle.updated)
-        # print(updated_bundle.version)
-        # print(updated_bundle.aliases)
-        assert updated_bundle.aliases[0] == 'ghi'
+        logger.info("..........Update that Bundle.................")
+        server_bdl.aliases = ['ghi']
+        update_data_bundle = server_bdl
+        update_response = self.request('UpdateDataBundle', data_bundle=update_data_bundle,
+                                       query={'data_bundle_id': data_bundle.id})
+        logger.info("..........Get that Bundle.................")
+        updated_bundle = self.request('GetDataBundle', data_bundle_id=update_response['data_bundle_id']).data_bundle
+        logger.info('updated_bundle.aliases: %r', updated_bundle.aliases)
+        logger.info('updated_bundle.updated: %r', updated_bundle.updated)
+        logger.info('data_bundle.aliases: %r', data_bundle.aliases)
+        logger.info('data_bundle.updated: %r', data_bundle.updated)
+        # logger.info(updated_bundle.version)
+        # logger.info(updated_bundle.aliases)
```

Comment:
https://github.com/ga4gh/data-object-service-schemas/blob/72b2c18d9bbfd68cea71eb9e79ee3819ca13c270/python/test/test_server.py#L318

Only leave this if they will be put on in the future, and add a TODO note.